pkgs/tools/misc/ppfetch/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ppfetch";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "yael333";
+    repo = "ppfetch";
+    rev = "acf00175113db7e4bcd04f40596494360d232fdf";
+    sha256 = "sha256-v1l+0MNY8aruoljf9jAzgrpgFiKgqdWTdLv4o78a8kU=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -Dm755 -t $out/bin ppfetch
+  '';
+
+  meta = with lib; {
+    description = " A colorful and prideful system info tool written in POSIX sh ";
+    homepage = "https://github.com/yael333/ppfetch";
+    license = licenses.mit;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ ];
```

Comment:
Would you be open to be a maintainer for this?
If yes please create a single commit to add yourself to the maintainers.list (see for an example)
and ensure that that commit comes before the ppfetch init commit in the commit history.